examples/create-new-project/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
The project path to add examples is 
```
module-ballerinax-jira/examples/<example-name>
```

Reply:
@Bhashinee I have created the example in the right place. Can you check again??

Reply:
I was referring to `<example-name>`. In your case, it was `create-new-project`. I tried to say that it should be the example name. Sorry for the confusion. The place is correct, it is just the name that you should update. :)